diff --git a/variety/plugins/builtin/downloaders/UnsplashDownloader.py b/variety/plugins/builtin/downloaders/UnsplashDownloader.py
index 07c1269..c878706 100644
--- a/variety/plugins/builtin/downloaders/UnsplashDownloader.py
+++ b/variety/plugins/builtin/downloaders/UnsplashDownloader.py
@@ -34,6 +34,7 @@ class UnsplashDownloader(SimpleDownloader):
         b"AwBXAAFSUQtSAAUGBQQED11dBAdRDQMFVQwCVgAOBQwCDwMDDAYDBARTAgIGAVdVCQdRBQUCU1NV\n"
         b"DARSAQgBWA==\n"
     )
+    USER_API_KEY = "$UNSPLASH_API_KEY"
     UTM_PARAMS = "?utm_source=Variety+Wallpaper+Changer&utm_medium=referral"

     rate_limiting_started_time = 0
@@ -66,11 +67,11 @@ class UnsplashDownloader(SimpleDownloader):
         return "unsplash_v2"

     def get_default_throttling(self):
-        return Throttling(max_downloads_per_hour=10, max_queue_fills_per_hour=1)
+        return Throttling(max_downloads_per_hour=50, max_queue_fills_per_hour=20)

     def get_unsplash_api_url(self):
         return "https://api.unsplash.com/photos/random?count=30&client_id={}{}".format(
-            Util.unxor(UnsplashDownloader.HASH, UnsplashDownloader.API_KEY),
+            UnsplashDownloader.USER_API_KEY,
             "&orientation=landscape" if self.get_variety().options.use_landscape_enabled else "",
         )

@@ -85,7 +86,7 @@ class UnsplashDownloader(SimpleDownloader):
         logger.info(lambda: "Filling Unsplash queue from " + url)

         r = Util.request(url)
-        if int(r.headers.get("X-Ratelimit-Remaining", 1000000)) < 1000:
+        if int(r.headers.get("X-Ratelimit-Remaining", 1000000)) < 100:
             UnsplashDownloader.rate_limiting_started_time = time.time()

         queue = []
